django_rdkit/models/fields.py

```diff
@@ -206,8 +247,16 @@ class MolLookupMixin:
     def get_prep_lookup(self):
```

Comment:
@MichelML I agree and I am also not particularly happy with having the conditional logic over the selected serialization strategy implemented in two places, but the `Field` class implements a different `get_prep_lookup`. I'll briefly explore using delegation instead of inheritance.